manager/assets/modext/widgets/resource/modx.grid.trash.js

```diff
@@ -469,6 +469,7 @@ Ext.extend(MODx.grid.Trash, MODx.grid.Grid, {
                 + ((record.json.longtitle) ? '<p><strong>' + _('long_title') + ':</strong> ' + record.json.longtitle + '</p>' : '')
                 + ((record.data.parentPath) ? '<p><strong>' + _('trash.parent_path') + ':</strong> ' + record.data.parentPath + '</p>' : '')
                 + ((record.json.content) ? '<p><strong>' + _('content') + ':</strong> ' + Ext.util.Format.ellipsis(record.json.content.replace(/<\/?[^>]+>/gi, ''), 100) + '</p>' : '');
+            preview = Ext.util.Format.htmlEncode(preview);
```

Comment:
When merging, please double check that this does not cause all the `<p><strong>`'s to be rendered as text. From reading the code I don't think it will, as the qtip has to be added into the div first and presumably that will decode the html to render it appropriately, but just for making sure.

Reply:
Done! Works as described 👍